dwave/optimization/symbols.pyx

```diff
@@ -1670,6 +1672,41 @@ cdef class Equal(ArraySymbol):
 
 _register(Equal, typeid(cppEqualNode))
 
+cdef class Expit(ArraySymbol):
+    """Takes the values of a symbol and returns the corresponding logistic sigmoid (expit).
+
+    Examples:
+        This example adds the expit of a decision
+        variable to a model.
+
+        >>> from dwave.optimization.model import Model
+        >>> from dwave.optimization.mathematical import expit
+        >>> model = Model()
+        >>> i = model.constant(1.6)
+        >>> ii = expit(i)
+        >>> type(ii)
+        <class 'dwave.optimization.symbols.Expit'>
+    """
```

Comment:
```suggestion
    """Takes the values of a symbol and returns the corresponding logistic sigmoid (expit).

    See Also:
        :func:`~dwave.optimization.mathematical.expit`: equivalent function.
        
    .. versionadded:: 0.5.2
    """
```
What you had was consistent with other symbols in this file, so apologies for changing the pattern out from under you.

We want to discourage folks from using the symbol classes directly, so I think in general we just want to redirect users to the function/method that constructs it.

We also want to add the `..versionadded` when possible. The rule-of-thumb for that is to just increment the patch number. We'll go back an edit them if we end up changing the minor or major.

Reply:
That makes sense. Done. Thanks!